### ISSUE TITLE
INSPIRE EPSG Code in datametadata vs. servicemetadata

### ISSUE THREAD

**Issue by Michael-81** [inspire-eu-validation/ets-repository#135]
For Metadatatest:

**LINK to Report:**   http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EID6f536993-4daa-4fb2-b86d-68e9de9436e4.html?lang=en

> The metadata record set has 1 record(s) with errors for this assertion.
> XML document 'GetRecordByIdServant$Get.xml', record 'ff8081812d73f5f3012d933aefd80136': The metadata record references none of the expected coordinate reference systems. The following reference system code have been referenced: '31467'. Please refer to table 2 of the data specification for the list of expected coordinate reference system codes.

INSPIRE EPSG Code for DataMetadata:

!! It's not mandatory to define an INSPIRE EPSG Code in the datametadata !! Only the service metadata needs INSPIRE EPSG Codes!

Example: on-the-fly-transformation dataset is EPSG:25832 with transformation by geoserver to EPSG:4336 - only written in the servicemetadata!

**Comment by cportele** [comment]
I agree. The metadata for interoperability requirements are met with any CRS.

Assertion `md-iop.a.1` needs to be changed to remove the part that returns the message `TR.unknownCRS3`.
